nixio.spec.in

```diff
@@ -5,9 +5,13 @@ Summary:	IO-libray for nix data files
 
 License:	BSD
 URL:		https://www.g-node.org/nix
-Source0:	%{name}-%{version}.tar.gz
+Source0:	https://github.com/G-Node/nix/archive/%{version}/%{name}-%{version}.tar.gz
```

Comment:
if I am not mistaken then the tar url is https://github.com/G-Node/nix/archive/%{version}.tar.gz

Reply:
hm, not sure, If I try curl on the urls, it does not work. ``wget https://github.com/G-Node/nix/archive/1.4.3/1.4.3.tar.gz`` works but is redirected to `` https://codeload.github.com/G-Node/nix/tar.gz/1.4.3`` so ``curl https://codeload.github.com/G-Node/nix/tar.gz/1.4.3 -o 1.4.3.tar.gz`` succeeds.
So, imho, the ``%{name}-%{version}.tar.gz`` part will not work.

Reply:
Edit the generated spec-file to make the url point to something valid:
```
--- nixio.spec	2019-01-06 13:04:15.465030789 +0100
+++ nixio-1.4.3.spec	2019-01-06 13:04:01.057849027 +0100
@@ -1,5 +1,5 @@
 Name:		nixio
-Version:	1.5.0
+Version:	1.4.3
 Release:	0.20190104gitcc8d3764%{?dist}
 Summary:	IO-libray for nix data files
```
Then use `spectool` to try to download it directly from the specfile:
```
spectool -g -R -C . nixio.spec
Getting https://github.com/G-Node/nix/archive/1.4.3/nixio-1.4.3.tar.gz to ./nixio-1.4.3.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   117    0   117    0     0    194      0 --:--:-- --:--:-- --:--:--   194
100  463k    0  463k    0     0   271k      0 --:--:--  0:00:01 --:--:-- 1672k
```